pkg/remote/fileshare/fsutil/fsutil.go

```
@@ -0,0 +1,316 @@
+	"log"
+	"regexp"
+	"strings"
+
+	"github.com/wavetermdev/waveterm/pkg/remote/connparse"
+	"github.com/wavetermdev/waveterm/pkg/remote/fileshare/fstype"
+	"github.com/wavetermdev/waveterm/pkg/remote/fileshare/pathtree"
+	"github.com/wavetermdev/waveterm/pkg/util/utilfn"
+	"github.com/wavetermdev/waveterm/pkg/wshrpc"
+)
+
+var slashRe = regexp.MustCompile(`/`)
```

golangci-lint (1.62.2)

var `slashRe` is unused

(unused)